### DISCUSSION TITLE
Approved but no benefits received

### DISCUSSION THREAD

**Discussion by aangh182** [community/community#146210] [GitHub Education] [answered]
### Hi everyone,

It's been 12 days, and I still can't access my student pack. A few days ago, I also sent another academic affiliation request because I thought the first one wasn't approved. However, I don't know what happened to either of them.
<img width="351" alt="Screen Shot 2024-12-04 at 11 40 06" src="https://github.com/user-attachments/assets/56895175-c20e-49fa-b18a-033b57767aab">

**Comment by cachedcake** [comment, marked as answer]
Hi @aangh182 ,

Don't worry it's normal, when the benefits become available, the status label will turn purple and change to:

- Congrats! Your request for @aangh182 was approved and your benefits are now available.

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions:
[#132790](https://github.com/orgs/community/discussions/132790) , [#131665](https://github.com/orgs/community/discussions/131665), [#132470](https://github.com/orgs/community/discussions/132470), [#133531](https://github.com/orgs/community/discussions/133531), [#134279](https://github.com/orgs/community/discussions/134279), [#135849](https://github.com/orgs/community/discussions/135849), [#136887](https://github.com/orgs/community/discussions/136887), [#138721](https://github.com/orgs/community/discussions/138721), [#139268](https://github.com/orgs/community/discussions/139268), [#140287](https://github.com/orgs/community/discussions/140287), [#145640](https://github.com/orgs/community/discussions/145640)

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

Let me know of this helps!
